POLMED/src/styles/pages/_home.scss

```diff
@@ -14,18 +14,21 @@
 }
 
 .hp-up-column {
-  width: 25em;
+  width: min(100vw, 25em);
 }
 
 .hp-up-doctor {
   position: absolute;
   bottom: 0;
   max-height: 80vh;
+  max-width: 100vw;
 }
 
 .hp-up-title {
+  position: relative;
+  z-index: 1;
   color: white;
-  font-size: 4.5em;
+  font-size: clamp(1em, 10vw, 4.5em);
```

Comment:
co to znaczy clamp, bo nie wiem a jestem ciekaw

Reply:
W clampie definiujemy, idąc od lewej, minimalną, domyślną oraz maksymalną wartość. Bardzo przydatne do robienia responsywności.